src/css/neo4j-docs.css

```diff
@@ -0,0 +1,185 @@
+:root {
```

Comment:
I'm not a CSS expert but I think the variables should be defined in `vars.css`

Reply:
Where they are docs-only I was trying to keep them separate, but they can be moved.

Reply:
On second thought, it's probably better to let them here. I was suggesting to move everything in `vars.css` for consistency (because that's what we are currently doing) *but* I think your approach makes more sense.

Sorry :sweat_smile: